lib/V8/v8-globals.h

```diff
@@ -24,17 +24,22 @@
 #ifndef ARANGODB_V8_V8__GLOBALS_H
 #define ARANGODB_V8_V8__GLOBALS_H 1
 
-#include "Basics/Common.h"
+#include <string.h>
+#include <cstdint>
+#include <memory>
+#include <unordered_map>
+#include <utility>
+
+#include <v8.h>
```

Comment:
if you include v8 here, you can skip it from V8PlatformFeature (most probably)

Reply:
Better to just include it both places it's used. Keeps things more self-contained and reduces risk of further breakage in the future.